LBHackney-IT/lbh-fss-portal-frontend
# Add functionality to allow admin users to pause an organisation

**Description**

Give admins ability to pause organisations 

**Test instructions**

- pull this branch 
- navigate to /organisations
- check you are able to pause an organisation and receive a success message (any changes will not take effect as using mock api)
- open /mock-api/organisations/__organisationId__/PATCH.js and return a 400 rather than 200 response, then try to pause an organisation and check you receive a fail message 

![image](https://user-images.githubusercontent.com/69238887/101469124-38e0a700-393c-11eb-9d6b-26e7374b99df.png)


**Ticket**
https://app.clickup.com/t/apyehp
